pandas/tests/io/test_parquet.py

```diff
@@ -977,18 +976,6 @@ def test_timestamp_nanoseconds(self, pa):
     def test_timezone_aware_index(self, request, pa, timezone_aware_date_list):
         pytest.importorskip("pyarrow", "11.0.0")
 
-        if (
-            timezone_aware_date_list.tzinfo != datetime.timezone.utc
-            and pa_version_under18p0
```

Comment:
This was added in the PR removing our default usage of pytz. @mroeschke you were assuming that pyarrow would have fixed this in time for 18.0?

Reply:
(I changed the test to test the actual (although undesired) behaviour, just to ensure we are still testing the behaviour in general, and then this should also start failing once pyarrow fixes it, just like with the xfail)

Reply:
> you were assuming that pyarrow would have fixed this in time for 18.0?

I don't recall anymore; this might been an unnecessary "is pyarrow installed" check.